test/Makefile

```diff
@@ -88,10 +89,14 @@ coverage: check-go-version $(COVERAGE_BINARIES) $(TEST_PAYLOAD)
 	pkill -9 piggie; \
 	}
 	cd crit/ && GOCOVERDIR=${COVERAGE_PATH} ./crit-test.coverage
+	$(MAKE) -C ../crit/ unit-test GOFLAGS="-coverprofile=${COVERAGE_PATH}/coverprofile-crit-unit-test"
+	$(MAKE) -C crit/ e2e-test GOCOVERDIR=${COVERAGE_PATH}
 	$(MAKE) -C crit/ clean
 	# Print coverage from this run
 	$(GO) tool covdata percent -i=${COVERAGE_PATH}
 	$(GO) tool covdata textfmt -i=${COVERAGE_PATH} -o ${COVERAGE_PATH}/coverage.out
+	cat .coverage/coverprofile* | \
```

Comment:
Just a question if this is necessary with Go 1.20? Seeing this seems to imply it is necessary, but just asking to be sure if this is really necessary.

Reply:
Yes, the test coverage from `go test` is stored in `coverprofile-crit-unit-test` but with `codecov` we upload only `coverage.out`. This is necessary for the coverage results to show up in GitHub.